lib/ansible/modules/remote_management/manageiq/manageiq_provider.py

```diff
@@ -37,7 +37,7 @@
   type:
     description: The provider's type.
     required: true
-    choices: ['Openshift', 'Amazon', 'oVirt']
+    choices: ['Openshift', 'Amazon', 'oVirt', 'VMware']
```

Comment:
Is there a way to document VMware is new in 2.5? (oVirt might be too BTW)

Reply:
Not sure, I thought adding the example config was sufficient but let me know if there is some other way.